packages/backend/src/clients/Aws/s3.ts

```diff
@@ -81,7 +81,7 @@ export class S3Service {
     }
 
     async uploadImage(image: Buffer, imageId: string): Promise<string> {
-        return this.uploadFile(imageId, image);
+        return this.uploadFile(`${imageId}.png`, image);
```

Comment:
What's the reason we add this extension now? Should we also add the mime type?

Reply:
slack is able to process image files without the file extension but the same doesn't happen when the user downloads this file from the browser. I assume the mime type is already in the image buffer 
cc @rephus

Reply:
Is the mime type in there @rephus ?